test/unittests/commands/validate_data/fixtures/validation_result.py

```diff
@@ -844,21 +884,20 @@ def result_unknown_component(onto_graph: Graph) -> tuple[Graph, Graph, Validatio
         sh:sourceShape api-shapes:rdfsLabel_Shape ;
         sh:value " " ] .
     """
-    validation_g = Graph()
-    validation_g.parse(data=validation_str, format="ttl")
     data_str = f"""{PREFIXES}
         <http://data/empty_label> a onto:ClassWithEverything ;
             rdfs:label " "^^xsd:string .
     """
-    onto_data_g = Graph()
-    onto_data_g += onto_graph
-    onto_data_g.parse(data=data_str, format="ttl")
-    val_bn = next(validation_g.subjects(RDF.type, SH.ValidationResult))
+    graphs = Graph()
+    graphs.parse(data=validation_str, format="ttl")
+    graphs.parse(data=data_str, format="ttl")
+    graphs += onto_graph
+    val_bn = next(graphs.subjects(RDF.type, SH.ValidationResult))
     base_info = ValidationResultBaseInfo(
         result_bn=val_bn,
         source_constraint_component=SH.PatternConstraintComponent,
         resource_iri=DATA.empty_label,
         res_class_type=ONTO.ClassWithEverything,
         result_path=RDFS.label,
     )
-    return validation_g, onto_data_g, base_info
+    return graphs, base_info
```

Comment:
Some fixtures are used, where the distinction between the graphs is necessary. I changed those where the distinction is not relevant.